pkg/entity/user.go

```diff
@@ -130,3 +131,11 @@ func (u *User) RealName() string {
 	}
 	return ""
 }
+
+func (u *User) AvatarUrl() string {
+	if u.IsBot {
+		return "https://woogles-prod-assets.s3.amazonaws.com/macondog.png"
```

Comment:
this should probably be a temporary thing. I can upload the macondog avatar by signing in as HastyBot as well .. and perhaps other bots can have other avatars

Reply:
I added one commit to use the macondog only if the bot has no avatar_url in the DB.